src/App.css

```diff
@@ -345,7 +345,14 @@
 
```

Comment:
I highlighted in #147 that the CSS files are becoming too long and need to be segmented into multiple files. Approving for now but something to make a PR about for sure